web-ui/src/backup_account/backup_email/backup_email.spec.js

```diff
@@ -23,6 +24,15 @@ describe('BackupEmail', () => {
     expect(page.find('SubmitButton').props().buttonText).toEqual('backup-account.backup-email.button');
   });
 
+  it('form submit should call parameter custom submit', () => {
```

Comment:
Messages for tests should always start with a verb, since the syntax is: "BackupEmail calls...". Mocha follows the rule of `describe(<ComponentName>)`, `context(<Situation>)`, `it(<action>)`.